doc/articles/getting-started/counterapp/includes/include-mvux.md

```diff
@@ -13,35 +13,62 @@ As part of creating the application, we selected MVUX as the presentation framew
     }
     ```
 
-- Add the `Count` and `Step` properties, to the `MainModel` class. These fields both must be of type `IState<int>` and to initialize them we use `=> State.Value(...)`.
+- Add a new `partial record` above named `Countable`. This record will be responsible for updating our counter's properties all while ensuring immutability. Learn more about immutable records [here](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Mvux/WorkingWithRecords.html#how-to-create-immutable-records).
```

Comment:
```suggestion
- Add a new `partial record` above named `Countable`. This record will be responsible for updating our counter's properties all while ensuring immutability. Learn more about immutable records [here](xref:Uno.Extensions.Mvux.Records#how-to-create-immutable-records).
```

Reply:
@rajamatt  Make sure to verify that the link is properly working while running the doc locally please